discord_bots/cogs/admin.py

```diff
@@ -0,0 +1,692 @@
+            )
+        )
+        os.execv(sys.executable, ["python", "-m", "discord_bots.main"])
+
+    @group.command(name="setbias", description="Set team bias")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(member="Member to bias", amount="Bias value")
+    async def setbias(self, interaction: Interaction, member: Member, amount: float):
+        if amount < -100 or amount > 100:
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Amount must be between -100 and 100",
+                    colour=Colour.red(),
+                ),
+                ephemeral=True,
+            )
+            return
+        await interaction.response.send_message(
+            embed=Embed(
+                description=f"Team bias for {member.name} set to `{amount}%`",
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(name="setcaptainbias", description="Set captain bias")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(member="Member to bias", amount="Bias value")
+    async def setcaptainbias(
+        self, interaction: Interaction, member: Member, amount: float
+    ):
+        if amount < -100 or amount > 100:
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Amount must be between -100 and 100",
+                    colour=Colour.red(),
+                ),
+                ephemeral=True,
+            )
+            return
+        await interaction.response.send_message(
+            embed=Embed(
+                description=f"Captain bias for {member.name} set to `{amount}%`",
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(
+        name="setcommandprefix", description="Sets the prefix for context commands"
+    )
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(prefix="New command prefix")
+    async def setcommandprefix(self, interaction: Interaction, prefix: str):
+        # TODO move to db-config
+        global COMMAND_PREFIX
+        COMMAND_PREFIX = prefix
+        await interaction.response.send_message(
+            embed=Embed(
+                description=f"Command prefix set to {COMMAND_PREFIX}",
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(name="unban", description="Unban player")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(member="Player to be unbanned")
+    async def unban(self, interaction: Interaction, member: Member):
+        session: SQLAlchemySession
+        with Session() as session:
+            players = session.query(Player).filter(Player.id == member.id).all()
```

Comment:
same as before: use `.first()`

Reply:
Updated to use `.first()`, along with the subsequent logic.